src/store/modules/gateway/types.ts

```diff
@@ -0,0 +1,6 @@
+import { AppGatewayServiceClient, grpc } from '../../types';
+
+export interface ModuleState {
+  grpcClient: AppGatewayServiceClient;
+  httpHeader: grpc.Metadata;
+}
```

Comment:
Can we just get rid of this file and move the interface to `state.ts` just like `ActionContext` in `actions.ts`? This is just an extra file.

Reply:
I meant to delete this file. It's already moved to root `types.ts`